### ISSUE TITLE
Clarification on mcd-unet-train.py

### ISSUE THREAD

**Issue by anikat1** [nchlis/keras_UNET_segmentation#1]
The script and from the developer's description dropout layer has been only used in test and not while training. In Line 146 of mcd-unet-train the training parameter for dropout is set to false while in mcd-unet-eval it is set to true.

De-vries paper code is adapted from also did not mention anywhere dropout should be inactive in training. Is there a particular reason?
